man/systemd.network.xml

```diff
@@ -1091,7 +1091,9 @@
             <literal>blackhole</literal> packets are discarded silently,
             <literal>unreachable</literal> packets are discarded and the ICMP message host unreachable is generated,
             <literal>prohibit</literal> packets are discarded and the ICMP message communication administratively
-            prohibited is generated. Defaults to <literal>unicast</literal>.
+            prohibited is generated,
+            <literal>throw</literal> lookup in current routing table fails and routing selection process returns to RPDB.
+            Defaults to <literal>unicast</literal>.
```

Comment:
hmm, urks, so the current paragraph is pretty badly written, we should really fix that. Now i figure we can leave that as is in the PR, but your addition really should be clean. Maybe write this as sentence of its own:

`If <literal>throw</literal> the lookup in the current routing table will fail, … .` 

or so. Also, what is RPDB? Acronyms like that really should have a short explanation, as I really don't know what that is... and if i don't know I am pretty sure not many others will know...

Reply:
RPDB stands for [Routing Policy Database](http://linux-ip.net/html/routing-rpdb.html).

Maybe we should write out the abbreviation here.